tzbuddy/__init__.py

```diff
@@ -57,7 +59,7 @@ def reordered(tzs, inverse):
     return reversed([x[0] for x in st])
 
 
-def calculate_tz(date, tz, span=24, header=True, sep=True):
+def calculate_tz(date, tz, hour_format_12, span=24, header=True, sep=True):
```

Comment:
I would have preferred to have a default value of `False` for this new parameter, but I guess we can live with this.